python/docs/source/examples/time_centric.ipynb

```diff
@@ -173,12 +173,8 @@
    "id": "c2c5a298",
    "metadata": {},
    "source": [
-    "Kaskada query language is parsed by the `fenl` extension. Query calculations are\n",
-    "defined in a code blocks starting with `%%fenl`.\n",
-    "\n",
-    "See [the `fenl`\n",
-    "documentation](https://kaskada-ai.github.io/docs-site/kaskada/main/fenl/fenl-quick-start.html)\n",
-    "for more information.\n",
+    "Kaskada queries are defined in Python, using the `{py}Timestream` class.\n",
+    "Sources are Timestreams generally containing [records](../guide/data_types.md#record-types).\n",
```

Comment:
Is there a scenario where we wouldn't interpret a `Source` input as a `record`?

Reply:
Conceivably. If we eventually added a timeseries / sensor input, I could see a case where we had something like a `timestream of f64` (sensor readings).